carbon/components/cards/tokenDetails/TokenVolumeOverTimeCard/index.tsx

```diff
@@ -29,16 +30,25 @@ export default function TokenVolumeOverTimeCard(
     />
   );
 }
-
 /** Async server component that renders a Recharts client component */
 async function TokenVolumeOverTimeChart(props: TokenDetailsProps) {
```

Comment:
In this chart we show a white background which is not in line with the designs where we show full bars:

Current:

<img width="552" alt="grafik" src="https://github.com/KlimaDAO/klimadao/assets/138768831/5a853e05-d6da-470b-802d-bd1f6e2ed854">

Designs:

<img width="312" alt="grafik" src="https://github.com/KlimaDAO/klimadao/assets/138768831/63c0b1f8-4eb0-422c-ac2a-463ba32723a3">

Reply:
Also notice the colors, they should be more contrasty (blue from the alternate palette)

Reply:
Hi @jabby09 @theprofessora, I went a little bit freestyle on the token details charts because I saw that they were copy/paste of each other. I took inspiration from the renders on the site currently on production. Can you give your feedback? should I try to adhere to the figma designs?
![image](https://github.com/KlimaDAO/klimadao/assets/11857992/175ac8fe-c0f5-4cef-b4f8-bb93fa2a1ef9)